src/Symfony/Bundle/FrameworkBundle/Command/KernelDebugCommand.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * A console command to display kernel information.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+class KernelDebugCommand extends ContainerAwareCommand
+{
+    /**
+     * @var string
+     */
+    private $baseDir;
```

Comment:
I don't think this prop is legitimate. Looking at the code, you should inject the value as an argument to formatPath instead